indico/modules/events/models/events_test.py

```diff
@@ -123,3 +125,30 @@ def test_iter_days_dst(dummy_event, tz, start_time, end_time, start_date, end_da
     all_days = list(dummy_event.iter_days())
     assert all_days[0] == days[0]
     assert all_days[-1] == days[-1]
+
+
+@pytest.mark.parametrize(('supported_locales', 'default_locale', 'user_lang', 'force_language', 'locale'), (
+    # Test with user who has force_language setting enabled
+    (['en_US', 'en_GB', 'de_DE'], 'de_DE', 'fr_FR', True, 'fr_FR'),
+    # Test with user who has force_language setting disabled and supported language
+    (['en_US', 'en_GB', 'de_DE'], 'de_DE', 'en_US', False, 'en_US'),
+    # Test with user who has force_language setting disabled and unsupported language
+    (['en_US', 'en_GB', 'de_DE'], 'de_DE', 'fr_FR', False, 'de_DE'),
+))
+def test_force_event_locale(db, dummy_event, supported_locales, default_locale, user_lang, force_language, locale):
+    # Setup Event
+    dummy_event.supported_locales = supported_locales
+    dummy_event.default_locale = default_locale
+
+    # Setup User
+    user = User(id=123, first_name='Guinea', last_name='Pig', email='guinea@pig.com')
+    db.session.add(user)
```

Comment:
We have a `create_user` fixture for this (or use the existing `dummy_user` via the fixture with the same name)